### DISCUSSION TITLE
How to get insight of when the card moved between column status in Github Project Beta using API ?

### DISCUSSION THREAD

**Discussion by ayyoubmaulana** [community/community#10024] [Projects and Issues]
Currently we use project beta to manage our task. But we need to get some insight like when the card issues moved between column status, for example card A moved from To do to In Progress at '2022-01-13T00:00:00Z'. I want to get that date using project beta API

**Comment by richnsoos** [comment]
@ayyoubmaulana , have you found a solution that enables you to get insight of when a card moves between columns in GH projects using API?
